docs/advanced-topics/wes/wes-cwl-agc-tutorial.rst

```diff
@@ -0,0 +1,247 @@
+
+1. Deploying an AGC project and context
+2. Configuring the Dockstore CLI to communicate with AGC infrastructure
+3. Launching a workflow
+
+
+Configuring AGC and the Dockstore CLI
+----------------------------------------
+1. Create a file named ``agc-project.yaml`` that contains:
+
+    .. code:: text
+
+        name: dockstoreAgcTutorialProject
+        schemaVersion: 1
+        contexts:
+          ctx2:
+            engines:
+              - type: cwl
+                engine: toil
+
+This will create an AGC project named ``dockstoreAgcTutorialProject``, with a single context named ``ctx2``.
+
+.. note::
+
+    For AGC infrastructure to interact with an S3 resource, the desired S3 bucket must be specified in the project's ``agc-project.yaml`` file
+    and your AWS account must already have access to the S3 resource. For more information on how to this, please click `here <https://aws.github.io/amazon-genomics-cli/docs/concepts/data/>`_.
+
+
+2. Activate AGC on your account. If this is your first time running AGC on an account, this may take a few minutes.
+
+    .. code:: text
+
+        agc account activate
+
+3. Deploy an AGC context by running the below command in the same directory as ``agc-project.yaml``. This may take a few minutes.
+
+    .. code:: text
+
+        agc context deploy ctx2
+
+4. Retrieve the WES endpoint created by the context. This will return a few values, the WES endpoint is the value under *WESENDPOINT*:
+
+    .. code:: text
+
+        agc context describe ctx2
+
+    .. code:: text
+
+        WESENDPOINT     https://example123.execute-api.us-west-2.amazonaws.com/prod/
+
+5. Copy the WES endpoint into the Dockstore CLI config file located at ``~/.dockstore/config`` and append ``ga4gh/wes/v1`` to the end of the URL.
+Your Dockstore CLI config file should have a named AWS profile included to allow the CLI to authorize requests to AWS. The resulting
+config file will look similar to:
+
+    .. code:: text
+
+            [WES]
+            url: https://example123.execute-api.us-west-2.amazonaws.com/prod/ga4gh/wes/v1
+            authorization: aws-wes-profile
+            type: aws
+
+6. To verify that the Dockstore CLI is communicating with the AGC infrastructure, list the WES server info. A JSON response will be printed
+to your terminal with the server's configuration.
+
+    .. code:: text
+
+        dockstore workflow wes service-info
+
+.. note::
+    At this point, the AGC infrastructure is deployed and the Dockstore CLI has been configured.
+
+    The AGC context and Dockstore configuration file do not need to be modified for the remainder of these examples, and will continue to function until the resources are modified and/or destroyed.
+
+
+Words Workflow
+--------------
+The Dockstore entry associated with this workflow can be found here `words <https://dockstore.org/workflows/github.com/dockstore-testing/tooltester-wes-testing/words:stable-version-for-testing-v4?tab=info>`_.
```

Comment:
I don't think you want `words`? Or is it some command?

Reply:
The workflow is called `words`, I got it from [here](https://github.com/aws/amazon-genomics-cli/tree/main/examples/demo-cwl-project/workflows/words), as it seemed like a good demo workflow.

Reply:
🤦